corehq/apps/events/tests/test_models.py

```diff
@@ -55,20 +55,20 @@ def get_attendee_cases(self):
             'Clarence Closedcase',
         ) as closed_case:
             self.factory.close_case(closed_case.case_id)
-            yield open_case, closed_case
+            yield open_case.case_id, closed_case.case_id
 
     def test_manager_returns_open_cases(self):
-        with self.get_attendee_cases() as (open_case, closed_case):
-            cases = [m.case for m in AttendeeModel.objects.by_domain(DOMAIN)]
-            self.assertEqual(cases, [open_case])
+        with self.get_attendee_cases() as (open_case_id, closed_case_id):
+            case_ids = [m.case_id for m in AttendeeModel.objects.by_domain(DOMAIN)]
+            self.assertCountEqual(case_ids, [open_case_id])
```

Comment:
Would it work to use `assertEqual` here?
```suggestion
            self.assertEqual(case_ids, [open_case_id])
```

Reply:
It would work, but I have a vague preference for the former, to keep it in line with the other test in the class, and because we're asserting that the two sequences contain the same elements, whereas `assertEqual` on lists _does_ imply that order matters.